frontend/src/modules/entitydetails/components/Metadata.css

```diff
@@ -49,3 +49,23 @@
     margin: 0;
     padding: 0;
 }
+
+.truncate {
+    text-overflow: ellipsis;
+    overflow: hidden;
+    padding-right: 50px;
```

Comment:
We could add a pixel or two more here, because sometimes ... gets really close to "See More". But we really need to make it dynamic, because the width will change with localizations. If that's too hard, we'll need to replace "See More" with "...".